src/app/core/components/search/search.component.html

```diff
@@ -9,5 +19,6 @@
       </a>
     </li>
   </ul>
-  <p *ngIf="options.length == 500">Only showing the first 500 results. Make your search term more concise.</p>
+  <p *ngIf="options.length == maxSearchResults">Only showing the first {{maxSearchResults}} results. Make your search
+    term more concise.</p>
```

Comment:
```suggestion
  <p *ngIf="options.length == maxSearchResults">Only showing the first {{maxSearchResults}} results. Make your search term more concise.</p>
```

Reply:
Why remove the line break, the formatter will put it back in?

Reply:
We have 80 line length? ...

Reply:
idk its a thing of html formatters